packages/js/components/src/media-uploader/stories/index.tsx

```diff
@@ -0,0 +1,164 @@
+		<>
+			{ render( {
+				open: () => setOpen( true ),
+			} ) }
+			{ isOpen && (
+				<Modal
+					title="Media Modal"
+					onRequestClose={ () => setOpen( false ) }
+				>
+					<p>
+						Use the default built-in{ ' ' }
+						<code>MediaUploadComponent</code> prop to render the WP
+						Media Modal.
+					</p>
+					{ Array( ...Array( 3 ) ).map( ( n, i ) => {
+						return (
+							<button
+								key={ i }
+								onClick={ () => {
+									onSelect( {
+										alt: 'Random',
+										url: `https://picsum.photos/200?i=${ i }`,
+									} );
+									setOpen( false );
+								} }
+								style={ {
+									marginRight: '16px',
+								} }
+							>
+								<img
+									src={ `https://picsum.photos/200?i=${ i }` }
+									alt="Random"
+									style={ {
+										maxWidth: '100px',
+									} }
+								/>
+							</button>
+						);
+					} ) }
+				</Modal>
+			) }
+		</>
+	);
+};
+
+const ImageGallery = ( { images }: { images: File[] } ) => {
+	return (
+		<div style={ { marginBottom: '16px' } }>
+			{ images.map( ( image, index ) => {
+				return (
+					<img
+						key={ index }
+						alt={ image.alt }
+						src={ image.url }
+						style={ {
+							maxWidth: '100px',
+							marginRight: '16px',
+						} }
+					/>
+				);
+			} ) }
+		</div>
+	);
+};
+
+const mockUploadMedia = ( { filesList, onFileChange } ) =>
+	new Promise< void >( ( resolve ) => {
+		const fileReader = new FileReader();
+		fileReader.onload = function ( event ) {
+			const file = {
+				alt: 'Temporary image',
+				url: event?.target?.result,
+			} as MediaItem;
+			onFileChange( [ file ] );
+			resolve();
+		};
+		fileReader.readAsDataURL( filesList[ 0 ] );
+	} );
+
+export const Basic: React.FC = () => {
+	const [ images, setImages ] = useState< File[] >( [] );
+
+	return (
+		<Card size="large">
+			<CardBody>
+				<ImageGallery images={ images } />
+				<MediaUploader
+					MediaUploadComponent={ MockMediaUpload }
+					onSelect={ ( file ) => setImages( [ ...images, file ] ) }
+					onError={ () => null }
+					onUpload={ ( files ) =>
+						setImages( [ ...images, ...files ] )
+					}
+					uploadMedia={ mockUploadMedia }
+				/>
+			</CardBody>
+		</Card>
+	);
+};
+
+export const DisabledDropZone: React.FC = () => {
+	const [ images, setImages ] = useState< File[] >( [] );
+
+	return (
+		<Card size="large">
+			<CardBody>
+				<ImageGallery images={ images } />
+				<MediaUploader
+					hasDropZone={ false }
+					label={ 'Click the button below to upload' }
+					MediaUploadComponent={ MockMediaUpload }
+					onSelect={ ( file ) => setImages( [ ...images, file ] ) }
+					onError={ () => null }
+					uploadMedia={ mockUploadMedia }
+				/>
+			</CardBody>
+		</Card>
+	);
+};
+
+export const MaxUploadFileSize: React.FC = () => {
+	const [ error, setError ] = useState< ErrorType | null >( null );
+
+	return (
+		<Card size="large">
+			<CardBody>
+				{ error && (
+					<Notice status={ 'error' }>{ error.message }</Notice>
```

Comment:
This story's error message doesn't dismiss on clicking the X icon. I don't know if this is just a problem in the Storybook context or if Notice doesn't provide a default that works....

Reply:
Removed the dismissable option for the notice.